From a95a727fdcd110470d99bd431129ec1624881c7f Mon Sep 17 00:00:00 2001
From: "dyoung@redhat.com" <dyoung@redhat.com>
Date: Mon, 31 Aug 2015 10:43:17 +0800
Subject: [PATCH] Revert "systemd/emergency.service: do not run for
 action_on_fail=continue"

This reverts commit dcae873414ff643e1de790f256e414923e2aef8b.

action_on_fail is useless thus remove the commits for it one by one.

Signed-off-by: Dave Young <dyoung@redhat.com>
---
 modules.d/98dracut-systemd/emergency.service | 1 -
 1 file changed, 1 deletion(-)

diff --git a/modules.d/98dracut-systemd/emergency.service b/modules.d/98dracut-systemd/emergency.service
index c19fe37..35d9c31 100644
--- a/modules.d/98dracut-systemd/emergency.service
+++ b/modules.d/98dracut-systemd/emergency.service
@@ -12,7 +12,6 @@ Description=Emergency Shell
 DefaultDependencies=no
 After=systemd-vconsole-setup.service
 Wants=systemd-vconsole-setup.service
-ConditionKernelCommandLine=!action_on_fail=continue
 
 [Service]
 Environment=HOME=/
